backend/src/bundles/subscription-plans/subscription-plan.service.ts

```diff
@@ -0,0 +1,21 @@
+import { type Service } from '~/common/types/types.js';
+
+import { type SubscriptionPlanRepository } from './subscription-plan.repository.js';
+import { type SubscriptionPlansGetAllResponseDto } from './types/types.js';
+
+class SubscriptionPlanService
+    implements Omit<Service, 'find' | 'create' | 'update' | 'delete'>
```

Comment:
You can just avoid implementing if you are sure that methods from the interface are not needed